### PR TITLE
PAG-717 add mock ec

### PR DESCRIPTION
**Pull request by pasqualedevita**
### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/api/product-mock-ec/api-mock-ec/v1/_base_policy.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - To remove a policy, delete the corresponding policy statement from the policy document.
+    - Position the <base> element within a section element to inherit all policies from the corresponding section element in the enclosing scope.
+    - Remove the <base> element to prevent inheriting policies from the corresponding section element in the enclosing scope.
+    - Policies are applied in the order of their appearance, from the top down.
+    - Comments within policy elements are not supported and may disappear. Place your comments between policy elements or at a higher level scope.
+-->
+<policies>
+    <inbound>
+        <base />
+        <rewrite-uri template="/" />
```
Comment (pasqualespica) [inline src/api/product-mock-ec/api-mock-ec/v1/_base_policy.xml:16]: @pasqualedevita without modifying the app through the policy I can remove the path in request. So it goes 🚀 

```sh
curl --location --request POST 'https://api.dev.platform.pagopa.it/mock-ec/api/paVerifyPaymentNotice' \
--header 'Content-Type: application/xml' \
--data-raw '<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:nod="http://pagopa-api.pagopa.gov.it/node/nodeForPsp.xsd">
  <soapenv:Header />
  <soapenv:Body>
    <pafn:paVerifyPaymentNoticeReq>
      <idPA>77777777777</idPA>
      <idBrokerPA>77777777777</idBrokerPA>
      <idStation>77777777777_01</idStation>
      <qrCode>
        <fiscalCode>77777777777</fiscalCode>
        <noticeNumber>302989852145637231</noticeNumber>
      </qrCode>
    </pafn:paVerifyPaymentNoticeReq>
  </soapenv:Body>
</soapenv:Envelope>'
```sh

**Review by pasqualespica — DISMISSED** [review]
🚀

File: src/api/product-mock-ec/api-mock-ec/v1/_base_policy.xml (hunk 1 above)
Comment (pasqualedevita) [reply src/api/product-mock-ec/api-mock-ec/v1/_base_policy.xml:16]: Top! Before merge we can try to improve api and product dir

**Review by pasqualespica — APPROVED** [review]
🚀

**Pull request merged**